projects/extension/sql/incremental/009-drop-truncate-from-vectorizer-config.sql

```diff
@@ -2,5 +2,3 @@ DROP FUNCTION IF EXISTS ai.embedding_ollama(text,integer,text,boolean,jsonb,text
 DROP FUNCTION IF EXISTS ai.embedding_voyageai(text,integer,text,booleab,jsonb,text);
 DROP FUNCTION IF EXISTS ai.voyageai_embed(text,text,text,boolean,text,text);
 DROP FUNCTION IF EXISTS ai.voyageai_embed(text,text[],text,boolean,text,text);
-
-UPDATE ai.vectorizer SET config = config #- '{"embedding", "truncate"}' WHERE config @? '$.embedding.truncate';
```

Comment:
We can't delete this, can we? What happens to folks who install the extension to a version after this but before we dissociate the vectorizer?

Reply:
This code is only used to generate sql for upgrades to the next version (which divests already) so this is safe I think. We have no way to generate sql TO a version that's not the current version.